tests/test_extraction.py

```diff
@@ -5,16 +5,7 @@
 
 from invoice.invoice import populate_columns
 from invoice.load_data import read_invoice_file, read_server_info_file
-
-COLNAMES = [
-    "type",
-    "description",
-    "start_date",
-    "end_date",
-    "quantity",
-    "unit_price",
-    "price",
-]
+from invoice.values import COLNAMES
```

Comment:
Do you know if there is any PEP on the organization of python packages, that includes what you should do with such data? I think it is definitely more maintainable like that, just wondering out of interest if you know any resource that talks about that part.

Reply:
no, I don't know any PEP saying anything about it. I just though that extracting the value to a single point would avoid future errors. Some kind of https://en.wikipedia.org/wiki/Don%27t_repeat_yourself